src/components/e-date.vue

```diff
@@ -0,0 +1,326 @@
+           */
+          this.$emit('change', this.dateValue.toDate());
+        }
+      },
+
+      /**
+       * Handles updates on the value prop.
+       *
+       * @param {Date} date - The new date value.
+       */
+      value(date) {
+        this.dateValue = this.$dayjs(date);
+      },
+    },
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() {},
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      /**
+       * Event handler for 'enter' keypress on the input.
+       *
+       * @param {Event} event - The original DOM event.
+       */
+      onEnter(event) {
+        event.preventDefault(); // Prevents auto submit of form.
+        event.target.blur(); // Triggers update.
+      },
+
+      /**
+       * Event handler for the focus event of the input.
+       *
+       * @param {Event} event - The original DOM event.
+       */
+      onFocus(event) {
+        /**
+         * Forwards the focus event to the parent component.
+         *
+         * @event focus
+         *
+         * @property {Event} event - The original DOM event.
+         */
+        this.$emit('focus', event);
+      },
+
+      /**
+       * Event handler for the blur event of the input.
+       *
+       * @param {Event} event - The original DOM event.
+       */
+      onBlur(event) {
+        /**
+         * Forwards the blur event to the parent component.
+         *
+         * @event blur
+         *
+         * @property {Event} event - The original DOM event.
+         */
+        this.$emit('blur', event);
+      },
+    },
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  @use '../setup/scss/variables';
+  @use '../setup/scss/mixins';
+  @use '../setup/scss/extends';
+  @use '../setup/scss/functions';
+
+  .e-date {
+    $height: 48px;
+    $border: 1px;
+
+    display: flex;
+    align-items: center;
+    padding: rem(0 variables.$spacing--25); // Equalizes vertical centering
+    border: $border solid transparent;
+    border-radius: 99999px; // Force half circles.
+    background: variables.$color-grayscale--600;
+
+    @include mixins.media(lg) {
+      @include mixins.font(variables.$font-size--14, 18px);
+    }
+
+    &__label-text {
+      grid-area: label;
+      align-self: center;
+      color: variables.$color-primary--1;
+    }
+
+    &__input,
+    &__input-spacer {
+      margin: 0 variables.$spacing--5;
+      color: variables.$color-grayscale--200;
+    }
+
+    &__input {
+      @extend %input-reset;
+
+      grid-area: input;
+      width: 14ch; // 8 letters + 2 dividers + some, because IE11 does not count outer spacing of 0
+      height: rem($height - (2 * $border) - 1px);
+      margin-bottom: 1px; // Improves vertical centering.
+
+      @supports (display: inline-grid) {
+        width: var(--e-date-format-length, 10ch);
+      }
```

Comment:
todo: please replace the `width` from above with this and remove the `@support` wrapper. We don't support IE11 anymore.

Reply:
@patric-eberle What is "this" in our context ?